tests/roles/manila_adoption/tasks/main.yaml

```diff
@@ -0,0 +1,75 @@
+- name: deploy podified Manila with cephfs backend
+  when: manila_backend == "cephfs"
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    oc patch openstackcontrolplane openstack --type=merge --patch '
+    spec:
+      manila:
+        enabled: true
+        apiOverride:
+          route: {}
+        template:
+          databaseInstance: openstack
+          manilaAPI:
+            customServiceConfig: |
+              [DEFAULT]
+              enabled_share_protocols=nfs,cephfs
+            replicas: 1
+            networkAttachments:
+            - internalapi
+            override:
+              service:
+                internal:
+                  metadata:
+                    annotations:
+                      metallb.universe.tf/address-pool: internalapi
+                      metallb.universe.tf/allow-shared-ip: internalapi
+                      metallb.universe.tf/loadBalancerIPs: 172.17.0.80
+                  spec:
+                    type: LoadBalancer
+          manilaScheduler:
+            replicas: 1
```

Comment:
do we want 3 here and above? or is this test always going to run on a single node crc?

Reply:
right, the test is against a SNO/crc, I think we can keep `replicas:1` and propose a patch as long as we move to a multinode environment